src/components/toast/components/toaster/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Toaster as ReactHotToaster } from 'react-hot-toast'
+
+/**
+ * react-hot-toast's Toaster component, with predefined settings.
+ * Removes default toast styling from react-hot-toast,
+ * and manages styling within the Notification components.
+ *
+ * Note: largely duplicative of a similar component in @hashicorp/react-toast:
+ * https://github.com/hashicorp/react-components/blob/main/packages/toast/toaster/index.tsx
+ */
+export default function Toaster() {
+  return (
+    <ReactHotToaster
+      position="bottom-right"
+      gutter={16}
+      containerStyle={{
+        inset: 24,
+      }}
+      toastOptions={{
+        style: {
+          margin: 0,
+          padding: 0,
+          maxWidth: 404,
+          width: '100%',
+          backgroundColor: 'transparent',
+          boxShadow: 'none',
```

Comment:
It looks like in the hds spec there is a box-shadow 🤔 
<img width="533" alt="Screen Shot 2022-05-24 at 9 32 34 AM" src="https://user-images.githubusercontent.com/36613477/170087140-dccd546b-e1dd-497e-a4b4-77c3d6b64b83.png">